Sources/KIF/Additions/NSError-KIFAdditions.h

```diff
@@ -13,4 +13,6 @@
 + (instancetype)KIFErrorWithUnderlyingError:(NSError *)underlyingError format:(NSString *)format, ... NS_FORMAT_FUNCTION(2,3);
 + (instancetype)KIFErrorWithFormat:(NSString *)format, ... NS_FORMAT_FUNCTION(1,2);
 
++ (instancetype)KIFErrorFromException:(NSException *)exception;
```

Comment:
I think this got pulled over from the other PR. Worth reverting here, as it seems unused in this context.